websites/T/Trackmania.io/dist/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Greep",
+    "id": "330030648456642562"
+  },
+  "service": "Trackmania.io",
+  "altnames": [
+    "TM.io"
+  ],
+  "url": [
+    "trackmania.io",
+    "www.trackmania.io"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/W99K52i.jpg",
+  "thumbnail": "https://i.imgur.com/oCyBucc.jpg",
+  "color": "#0A7FB4",
+  "tags": [
+    "cars",
+    "racing",
+    "tm",
+    "tmio",
+    "tm2020",
+    "trackmania-io",
+    "trackmania-2020",
+    "trackmania",
```

Comment:
Unnecessary tags.

Reply:
i can remove some, but i think the "trackmania" tag is necessary

Reply:
Yup, you are totally right! Forgot to mention that you can leave it. Good point!